index.js

```diff
@@ -71,14 +71,14 @@ function findAllDependencies(file, knownDependencies, sourceDirectories, knownFi
 }
 
 // Given a source directory (containing top-level Elm modules), locate the
-// elm-package.json file that includes it and get all its source directories.
+// elm.json file that includes it and get all its source directories.
 function getElmPackageSourceDirectories(baseDir, currentDir) {
   if (typeof currentDir === "undefined") {
     baseDir = path.resolve(baseDir);
     currentDir = baseDir;
   }
 
-  var elmPackagePath = path.join(currentDir, 'elm-package.json');
+  var elmPackagePath = path.join(currentDir, 'elm.json');
```

Comment:
Not really `elmPackagePath` anymore, isn't it more like `elmJsonPath`?

Reply:
I guess it's still the file that defines the packages on which the elm build depends.
I'm happy to make the change to `elmJsonPath` if this terminology seems more sensible :)